src/components/data-processing/StatusIndicator.jsx

```diff
@@ -53,6 +55,12 @@ const StatusIndicator = () => {
       title: <Text strong type='danger'>failing</Text>,
       description: (
         <Text>
+          Reason:
+          {' '}
+          <Text type='danger'>
```

Comment:
There's no need to make this red. You can make the text `Reason` bold instead (I believe it's `<Text strong>Bold test</Text>`) and keep the reason normally formatted.